crates/matrix-sdk-crypto/src/store/crypto_store_wrapper.rs

```diff
@@ -68,6 +75,13 @@ impl CryptoStoreWrapper {
         let room_key_updates: Vec<_> =
             changes.inbound_group_sessions.iter().map(RoomKeyInfo::from).collect();
 
+        let withheld_session_updates: Vec<_> = changes
+            .withheld_session_info
+            .values()
+            .flat_map(|session_map| session_map.values())
+            .map(Clone::clone)
```

Comment:
Probably more Rust idiomatic to do as such:

```suggestion
            .flat_map(|session_map| session_map.values().cloned())
```